plugins/debuginfo-install.py

```diff
@@ -74,59 +76,170 @@ def configure(self):
         self.base.repos.enable_debug_repos()
 
     def run(self):
-        self.packages = self.base.sack.query()
-        self.packages_available = self.packages.available()
         errors_spec = []
 
+        debuginfo_suffix_len = len(Package.DEBUGINFO_SUFFIX)
+        debugsource_suffix_len = len(Package.DEBUGSOURCE_SUFFIX)
+
         for pkgspec in self.opts.package:
-            package_query = sorted(dnf.subject.Subject(pkgspec).get_best_query(
-                self.cli.base.sack).filter(arch__neq='src'), reverse=True)
-            if not package_query:
-                msg = _('No match for argument: %s')
-                logger.info(msg, self.base.output.term.bold(pkgspec))
+            solution = dnf.subject.Subject(pkgspec).get_best_solution(self.base.sack)
```

Comment:
Please add one key word:
`solution = dnf.subject.Subject(pkgspec).get_best_solution(self.base.sack, with_src=False)`
In some cases when glob is used in combination with parser it ciuld result that it returns only src packages and line below will remove them. I don't remember any example.

Reply:
Will do.